docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -724,7 +724,7 @@ Baseline coverage of treatment for elevated SBP and elevated LDL-c is substantia
     - `Validation workbook Model 2 <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Model2_VV_SBP.ipynb>`_ `And interactive sim <https://github.com/ihmeuw/vivarium_research_nih_us_cvd/blob/main/Interactive_Model2_VV.ipynb>`_ Cause model is identical to Model 1 with same pieces correct and the same discrepancies. Risk factors match for exposure, standard deviation and relative risk. Outstanding issue with individual simulant outliers in SBP and incidence. 
   * - 3.0
     - Adding angina as a cause    
-    - 
+    - `Validation workbook Model 3 <https://github.com/ihmeuw/vivarium_research_nih_us_cvd>`_ Cause model is identical to prior models with same pieces correct and the same discrepancies. Risk factors match for exposure, standard deviation and relative risk. Outstanding issue with individual simulant outliers in SBP and incidence. Seems that angina relative risk is highly susceptible to low n-size and leads to high variation. 
```

Comment:
Can you include the visual on the angina relative risk here?  There is so much stuff in the V&V notebooks that I think I missed this one.

Reply:
I added it! Good idea to paste it here